.gitmodules

```diff
@@ -1,7 +1,3 @@
-[submodule "models/sophiemeakin"]
-	path = models/sophiemeakin
-	url = git@github.com:sophiemeakin/ecdc-covid19-admissions.git
-	ignore = dirty
```

Comment:
do we not need `ignore = dirty` anymore? (not sure why it was there but possibly for a reason?)

Reply:
not sure either - it wasn't in the latest couple of modules added (CovidMetrics-epiBATS, ULZF)...

Reply:
ok, happy to try without and put back in if there's a problem